docs/core-concepts/issues/issue-types.md

```
@@ -103,6 +105,7 @@ Here's a list of all the property types and attributes that Plane provides for c
 | **Boolean**            | True/False                               | Default value is false. This attribute cannot be marked as mandatory.                                            |
 | **Date**               | Date Format                              | Consistent date format across all properties.                                                                    |
 | **Member&nbsp;picker** | Single Select, Multi select              | Displays a list of all project members.                                                                          |
+| **URL** | A URL field for linking to external resources. |
```

markdownlint-cli2 (0.21.0)

[warning] Table column count
Expected: 3; Actual: 2; Too few cells, row will be missing data

(MD056, table-column-count)